test cases/linuxlike/16 sanitizers/meson.build

```diff
@@ -0,0 +1,10 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright © 2023-2024 Intel Corporation
+
+project('sanitizer', 'c', meson_version : '>= 1.6')
+
+assert(get_option('b_sanitize') == 'address,undefined', 'order of address + undefined is not correct')
+
+array = get_option('b_sanitize', format : 2)
+assert('address' in array)
+assert('undefined' in array)
```

Comment:
This doesn't really assert what you want since `'foo' in 'foobar'` is valid.

Reply:
You're right. Of course, we don't have type information in the DSL (and I think that's good), so I've tried to come up with something better.